slownik.rst

```diff
@@ -0,0 +1,178 @@
+
+SaaS
+  Software as a Service (w skrócie SaaS, ang. oprogramowanie jako usługa) – jeden z modeli chmury obliczeniowej, w którym aplikacja jest przechowywana i wykonywana na komputerach dostawcy usługi i jest udostępniana użytkownikom przez Internet. Takie rozwiązanie eliminuje potrzebę instalacji i uruchamiania programu na komputerze klienta. Model SaaS przerzuca obowiązki instalacji, zarządzania, aktualizacji, pomocy technicznej z klienta na dostawcę usługi. W efekcie użytkownik oddaje kontrolę nad oprogramowaniem i obowiązek zapewnienia ciągłości jego działania dostawcy.
+
+SIEM
+  Security Information and Event Management — system, którego zadaniem jest zbieranie, korelowanie i alarmowanie w przypadku wykrycia incydentu bezpieczeństwa.
+
+SLA
+  Service Level Agreement — umowa o gwarantowanym poziomie świadczenia usług.
+
+SOC
+  Ang. Security Operation Center – zaawansowana struktura organizacyjna (personel, procedury i narzędzia analityczne), którą powołuje się w celu ochrony systemów informatycznych i sieciowych przed atakami w cyberprzestrzeni. Dobrze wyposażony i zorganizowany SOC prowadzi ochronę 24/7 w zakresie „Before, During and After&rdquo; (ang. „przed, w trakcie, po&rdquo;), czyli zabezpiecza systemy przed włamaniem, potrafi wykryć atak, gdy nastąpi i skutecznie go zablokować oraz potrafi zdiagnozować i usunąć skutki ataku.
+
+Sprawa
+  [za IK] zdarzenie lub stan rzeczy, w tym z zakresu postępowania administracyjnego, wymagające rozpatrzenia i podjęcia czynności służbowych lub przyjęcia do wiadomości.
+
+SSO
+  Single Sign On — mechanizm pozwalający na współdzielenie sesji użytkownika między aplikacjami.
+
+Thin provisioning
+  Wykorzystanie technologii wirtualizacji do zaprezentowania systemowi operacyjnemu większej liczby fizycznych zasobów niż się faktycznie posiada.
+
+Urzędnik
+  Osoba uprawniona, korzystająca z aplikacji.
+
+Uwierzytelnienie
+  Potwierdzenie, iż osoba zgłaszająca żądanie do aplikacji jest tożsama z osobą umieszczoną na liście uprawnionych.
+
+UX
+  ang. user experience - doświadczenie użytkownika, całość wrażeń, jakich doświadcza użytkownik podczas korzystania z produktu interaktywnego. W odniesieniu do projektowania oprogramowania chodzi o takie zaprojektowanie interfejsu, by był on dla użytkownika atrakcyjny, funkcjonalny, użyteczny, ergonomiczny a korzystanie z niego sprawiało przyjemność i dawało satysfakcję.
+
+WAF
+  ang. Web Application Firewall – zestaw reguł dotyczących transmisji HTTP; ochrona przed atakami takimi jak cross-site scripting (XSS) i SQL injection.
+
+Węzeł Krajowy
+  Rozwiązanie organizacyjno-techniczne, które zapewnia funkcjonowanie krajowego schematu identyfikacji elektronicznej oraz stanowi punkt przyłączenia akredytowanych systemów identyfikacji elektronicznej (DT) Węzeł krajowy umożliwia uwierzytelnienie w celu realizacji usług online z wykorzystaniem środka identyfikacji elektronicznej wydanego w ramach akredytowanego systemu identyfikacji elektronicznej.
```

Comment:
Brak kropki po `(DT)`. Czy `Węzeł Krajowy` to nazwa własna, której człony należy pisać z wielkiej litery?